cobbler.spec

```diff
@@ -317,6 +331,12 @@ ln -sf service %{buildroot}%{_sbindir}/rccobblerd
 # cobbler-web
 rm %{buildroot}%{_sysconfdir}/cobbler/cobbler_web.conf
 
+%if 0%{?suse_version}
+# cobbler-tests
+cp -r tests/ %{buildroot}/%{_datadir}/cobbler/
```

Comment:
This should be put into setup.py ?

Reply:
Nope. The tests should in a normal PyPi source installation never be installed. We don't want to deliver testcode unconditionally.

Reply:
But there seems to be an issue with `tests/`:
https://travis-ci.com/github/cobbler/cobbler/jobs/397392880#L4600

Reply:
Jep, @SchoolGuy and I are working on it.